js/flickity.js

```diff
@@ -560,6 +560,67 @@ Flickity.prototype.getAdjacentCellElements = function( adjCount, index ) {
   return cellElems;
 };
 
+/**
+ * Check and report the specified Cell is visible and returns false if the
+ * cell is completely hidden, otherwise:
+ *
+ *  - 'clippedLeft'
+ *  - 'clippedRight'
+ *  - true - fully visible
+ *
+ * @param cellIndex - index of the cell in the Flickity
+ * @returns {truthy} - that is false if not visible, truthy if visible
+ */
+Flickity.prototype.cellVisibility = function( cellIndex ) {
+  var value = false;
+  var cell = this.cells[ cellIndex ];
+  var cellViewportX;
+
+  if (cell) {
+    cellViewportX = cell.x + this.x;  // translate cell coordinates to viewport
```

Comment:
You can use `.getRestingPosition()`. See https://github.com/metafizzy/flickity/blob/v1.1.1/js/animate.js#L225